pylint/test/unittest_checker_refactoring.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 Alexander Todorov <atodorov@mrsenko.com>
+
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+"""Unittest for refactoring checker."""
+
+import astroid
```

Comment:
We usually prefer `functional` tests for checks, please check the `functional` directory for examples. We only write specific unittests if we can't test the new check with a functional one.